lib/apiservers/engine/backends/container.go

```diff
@@ -755,6 +754,374 @@ func (c *Container) imageExist(imageID string) (storeName string, err error) {
+	}
+	if info.ContainerConfig.AttachStderr != nil {
+		container.AttachStderr = *info.ContainerConfig.AttachStderr // Attach the standard error
+	}
+	if info.ContainerConfig.Tty != nil {
+		container.Tty = *info.ContainerConfig.Tty // Attach standard streams to a tty, including stdin if it is not closed.
+	}
+	if info.ContainerConfig.OpenStdin != nil {
+		container.OpenStdin = *info.ContainerConfig.OpenStdin // Open stdin
+	}
+	if info.ContainerConfig.ImageID != nil {
+		container.Image = *info.ContainerConfig.ImageID // Name of the image as it was passed by the operator (eg. could be symbolic)
+	}
+	if info.ContainerConfig.Labels != nil {
+		container.Labels = info.ContainerConfig.Labels // List of labels set to this container
+	}
+
+	// Fill in information about the process
+	if info.ProcessConfig.ExecArgs != nil {
+		container.Env = strings.Split(*info.ProcessConfig.ExecArgs, " ") // List of environment variable to set in the container
+	}
+	if info.ProcessConfig.ExecPath != nil {
+		container.Cmd = append(container.Cmd, *info.ProcessConfig.ExecPath) // Command to run when starting the container
+	}
+	if info.ProcessConfig.WorkingDir != nil {
+		container.WorkingDir = *info.ProcessConfig.WorkingDir // Current directory (PWD) in the command will be launched
+	}
+	//		container.Entrypoint      strslice.StrSlice     				// Entrypoint to run when starting the container
+
+	// Fill in information about the container network
+	if info.ScopeConfig == nil {
+		container.NetworkDisabled = true
+	} else {
+		container.NetworkDisabled = false
+		container.MacAddress = ""
+		container.ExposedPorts = nil  //FIXME:  Add once port mapping is implemented
+		container.PublishService = "" // Name of the network service exposed by the container
+	}
+
+	// Get the original container config from the image's metadata in our image cache.
+	var imageConfig *metadata.ImageConfig
+
+	if info.ContainerConfig.ImageID != nil {
+		imageConfig, _ = getImageConfigFromCache(*info.ContainerConfig.ImageID)
+	}
+
+	// Fill in the values with defaults from the original image's container config
+	// structure
+	if imageConfig != nil {
+		container.StopSignal = imageConfig.ContainerConfig.StopSignal // Signal to stop a container
+
+		container.OnBuild = imageConfig.ContainerConfig.OnBuild // ONBUILD metadata that were defined on the image Dockerfile
+
+		// Fill in information about the container's volumes
+		// FIXME:  Why does types.ContainerJSON have Mounts and also ContainerConfig,
+		// which also has Volumes?  Assuming this is a copy from image's container
+		// config till we figure this out.
+		container.Volumes = imageConfig.ContainerConfig.Volumes
+	}
+
+	return container
+}
+
+func networkFromContainerInfo(id string, info *models.ContainerInfo) *types.NetworkSettings {
+	if info == nil || info.ScopeConfig == nil {
+		return nil
+	}
+
+	networks := &types.NetworkSettings{
+		NetworkSettingsBase: types.NetworkSettingsBase{
+			Bridge:                 "",
```

Comment:
It's because I don't know what Docker returns for this yet.  I need to do a few docker containers with networks and see what they send back before I know what this value should be.

-